### PR TITLE
Adrian dev

### LINKED ISSUE #19: API Layer -  Streaming Availability
https://rapidapi.com/movie-of-the-night-movie-of-the-night-default/api/streaming-availability

* Create functions to find what streaming services a show is available on
* Cache data in Firebase to make sure we stay under the 100 requests/day limit as much as possible

Related: #18

### PR DESCRIPTION
**Pull request by adrian-jonsson-sjoedin**
Closes #19

### CODE REVIEW

**Review by hannesmann — APPROVED** [review]

**Comment by adrian-jonsson-sjoedin** [comment]
Closes #50

File: src/backend/model/personalListModel.tsx [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
+import availability from "../../backend/model/streamingAvailabilityDummyStargate";
+import react from "react"
+
+const dummyFirebaseMyList = ["tt0118480", "tt0118480"]
+ 
+const myListState = atom({
   key: "myList", // unique ID (with respect to other atoms/selectors)
   default: [], // default value (aka initial value)
 });
+export const selectedSeasonState = atom({
```
Comment (hannesmann) [inline src/backend/model/personalListModel.tsx:14]: I'm going to need this in the search results view too. I already added an atom to src/atoms.tsx but it's better if we keep it in the model. Should we put this in a shared file instead?

Comment (adrian-jonsson-sjoedin) [reply src/backend/model/personalListModel.tsx:14]: Hm I guess we could put all the atoms in their own file and just move it to the model. Doesn't matter much to me if it is in this file or the atom file

**Review by Anthial — APPROVED** [review]
LGTM

**Pull request merged**